config/initializers/nationalities.rb

```diff
@@ -1,231 +1,8 @@
-  ['XK', 'Kosovan'],
-  ['KW', 'Kuwaiti'],
-  ['KG', 'Kyrgyz'],
-  ['LA', 'Lao'],
-  ['LV', 'Latvian'],
-  ['LB', 'Lebanese'],
-  ['LR', 'Liberian'],
-  ['LY', 'Libyan'],
-  ['LI', 'Liechtenstein citizen'],
-  ['LT', 'Lithuanian'],
-  ['LU', 'Luxembourger'],
-  ['MO', 'Macanese'],
-  ['MK', 'Macedonian'],
-  ['MG', 'Malagasy'],
-  ['MW', 'Malawian'],
-  ['MY', 'Malaysian'],
-  ['MV', 'Maldivian'],
-  ['ML', 'Malian'],
-  ['MT', 'Maltese'],
-  ['MH', 'Marshallese'],
-  ['MQ', 'Martiniquais'],
-  ['MR', 'Mauritanian'],
-  ['MU', 'Mauritian'],
-  ['MX', 'Mexican'],
-  ['FM', 'Micronesian'],
-  ['MD', 'Moldovan'],
-  ['MC', 'Monegasque'],
-  ['MN', 'Mongolian'],
-  ['ME', 'Montenegrin'],
-  ['MS', 'Montserratian'],
-  ['MA', 'Moroccan'],
-  ['LS', 'Mosotho'],
-  ['MZ', 'Mozambican'],
-  ['NA', 'Namibian'],
-  ['NR', 'Nauruan'],
-  ['NP', 'Nepalese'],
-  ['NZ', 'New Zealander'],
-  ['NI', 'Nicaraguan'],
-  ['NG', 'Nigerian'],
-  ['NE', 'Nigerien'],
-  ['NU', 'Niuean'],
-  ['KP', 'North Korean'],
-  ['GB', 'Northern Irish'],
-  ['NO', 'Norwegian'],
-  ['OM', 'Omani'],
-  ['PK', 'Pakistani'],
-  ['PW', 'Palauan'],
-  ['PS', 'Palestinian'],
-  ['PA', 'Panamanian'],
-  ['PG', 'Papua New Guinean'],
-  ['PY', 'Paraguayan'],
-  ['PE', 'Peruvian'],
-  ['PN', 'Pitcairn Islander'],
-  ['PL', 'Polish'],
-  ['PT', 'Portuguese'],
-  ['GB', 'Prydeinig'],
-  ['PR', 'Puerto Rican'],
-  ['QA', 'Qatari'],
-  ['RO', 'Romanian'],
-  ['RU', 'Russian'],
-  ['RW', 'Rwandan'],
-  ['SV', 'Salvadorean'],
-  ['SM', 'Sammarinese'],
-  ['WS', 'Samoan'],
-  ['ST', 'Sao Tomean'],
-  ['SA', 'Saudi Arabian'],
-  ['GB', 'Scottish'],
-  ['SN', 'Senegalese'],
-  ['RS', 'Serbian'],
-  ['SL', 'Sierra Leonean'],
-  ['SG', 'Singaporean'],
-  ['SK', 'Slovak'],
-  ['SI', 'Slovenian'],
-  ['SB', 'Solomon Islander'],
-  ['SO', 'Somali'],
-  ['ZA', 'South African'],
-  ['KR', 'South Korean'],
-  ['SS', 'South Sudanese'],
-  ['ES', 'Spanish'],
-  ['LK', 'Sri Lankan'],
-  ['SH', 'St Helenian'],
-  ['LC', 'St Lucian'],
-  ['SD', 'Sudanese'],
-  ['SR', 'Surinamese'],
-  ['SZ', 'Swazi'],
-  ['SE', 'Swedish'],
-  ['CH', 'Swiss'],
-  ['SY', 'Syrian'],
-  ['TW', 'Taiwanese'],
-  ['TJ', 'Tajik'],
-  ['TZ', 'Tanzanian'],
-  ['TH', 'Thai'],
-  ['TG', 'Togolese'],
-  ['TO', 'Tongan'],
-  ['TT', 'Trinidadian'],
-  ['SH', 'Tristanian'],
-  ['TN', 'Tunisian'],
-  ['TR', 'Turkish'],
-  ['TM', 'Turkmen'],
-  ['TC', 'Turks and Caicos Islander'],
-  ['TV', 'Tuvaluan'],
-  ['UG', 'Ugandan'],
-  ['UA', 'Ukrainian'],
-  ['UY', 'Uruguayan'],
-  ['UZ', 'Uzbek'],
-  ['VA', 'Vatican citizen'],
-  ['VE', 'Venezuelan'],
-  ['VN', 'Vietnamese'],
-  ['VC', 'Vincentian'],
-  ['WF', 'Wallisian'],
-  ['GB', 'Welsh'],
-  ['YE', 'Yemeni'],
-  ['ZM', 'Zambian'],
-  ['ZW', 'Zimbabwean'],
-].freeze
+require 'dfe/reference_data/countries_and_territories'
 
+NATIONALITIES = DfE::ReferenceData::CountriesAndTerritories::COUNTRIES_AND_TERRITORIES.all.map do |n|
+  [n.id, n.id == 'GB' ? 'British' : n.citizen_names]
```

Comment:
The 'citizen names' for GB in this list is 'Briton, British', but we record it as 'British'